frontend/src/app/services/api.service.ts

```diff
@@ -140,11 +140,6 @@ export class ApiService {
         return this.http.post<AggregateResult[]>(url, data).toPromise();
     }
 
-    public wordcloudTasks(data: WordcloudParameters): Promise<TaskResult> {
```

Comment:
We may still bring back the feature to generate full word clouds, though. So perhaps don't get rid of the related tasks. Or is this now done via another mechanism?

Reply:
I think if we want to introduce this, it would be by extending this view instead:
https://github.com/UUDigitalHumanitieslab/I-analyzer/blob/842919a4b2bdaa0aee15c85cfb0d81d90ef4c05c/backend/download/views.py#L91-L114

But also, we use git for a reason: we can always revert the commit if we want to reuse this approach.

Reply:
In that case, this can indeed go.